app/src/test/java/dev/aurakai/auraframefx/ui/navigation/AppNavGraphTest.kt

```
@@ -0,0 +1,433 @@
+        composeTestRule.setContent {
+            navController = rememberNavController()
+            AppNavGraph(navController = navController)
+        }
+
+        val initialDestination = navController.currentDestination?.route
+
+        // Try to navigate to invalid destination
+        composeTestRule.runOnIdle {
+            try {
+                navController.navigate("invalid_destination")
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)